h/accounts/schemas.py

```diff
@@ -162,6 +162,16 @@ class RegisterSchema(CSRFSchema):
         ),
     )
 
+    comms_opt_in = colander.SchemaNode(
+        colander.Boolean(),
+        description=_("I would like to receive news about annotation and Hypothesis"),
```

Comment:
```suggestion
        description=_("I would like to receive news about annotation and Hypothesis."),
```

![Screenshot from 2020-08-12 09-22-19](https://user-images.githubusercontent.com/22498/89992561-572fe200-dc7d-11ea-82a9-06b0814271ea.png)

Reply:
I wonder whether "receive news" should say "receive email" as well, just to be clear. Right now the form doesn't actually say that they're consenting for us to _email_ them

Reply:
The text here was lifted directly from: https://github.com/hypothesis/product-backlog/issues/1086 (lack of full stop and all). Although I could have probably spotted that one.